motto/agents/anthropic_agent.py

```diff
@@ -0,0 +1,51 @@
+import anthropic
+from .agent import Agent,Response
+import httpx
+import os
+import logging
+import time
+
+
+# FIXME: A more flexible was to setup proxy?
+proxy=os.environ.get('OPENAI_PROXY')
```

Comment:
Is the proxy relevant for AnthropicAgent?

Reply:
I'm not sure. It is a question to you and @Necr0x0Der. Do we need proxy for Anthropic in Russia?

Reply:
If Anthropic supports httpx client with proxy, then if proxy is set, why not to use it? But maybe we need to rename `OPENAI_PROXY` to `HTTP_PROXY`?

Reply:
This can be done in another PR, though